data-serving/data-service/src/controllers/case.ts

```diff
@@ -213,15 +213,15 @@ export class CasesController {
         cases: any[],
     ): Promise<BatchValidationErrors> => {
         const errors: { index: number; message: string }[] = [];
-        await Promise.all(
-            // We're about to validate this data; any is fine, here.
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            cases.map((c: any, index: number) => {
-                return new Case(c).validate().catch((e) => {
-                    errors.push({ index: index, message: e.message });
-                });
-            }),
-        );
+        // Do not parallelize these requests as it causes an out of memory error
+        // for a large number of cases. However this does take a long time to run
+        // in parallel, so if Mongo creates a batch validate method that should be used here.
```

Comment:
s/in parallel/sequentially? unless I am insunderstanding this comment...

Reply:
yes, good catch, changed.